core/app/service/backup.go

```diff
@@ -489,87 +496,59 @@ func (u *BackupService) checkBackupConn(backup *model.BackupAccount) (bool, erro
 		varMap := make(map[string]interface{})
 		if err := json.Unmarshal([]byte(backupItem.Vars), &varMap); err != nil {
-			global.LOG.Errorf("Failed to refresh OneDrive token, please retry, err: %v", err)
-			return
+			global.LOG.Errorf("Failed to refresh %s - %s token, please retry, err: %v", backupItem.Type, backupItem.Name, err)
+			continue
+		}
+		var (
+			refreshToken string
+			err          error
+		)
+		switch backupItem.Type {
+		case constant.OneDrive:
+			refreshToken, err = client.RefreshToken("refresh_token", "refreshToken", varMap)
+		case constant.GoogleDrive:
+			refreshToken, err = client.RefreshGoogleToken("refresh_token", "refreshToken", varMap)
+		case constant.ALIYUN:
+			refreshToken, err = client.RefreshALIToken(varMap)
 		}
-		refreshToken, err := client.RefreshToken("refresh_token", "refreshToken", varMap)
-		varMap["refresh_status"] = constant.StatusSuccess
-		varMap["refresh_time"] = time.Now().Format(constant.DateTimeLayout)
 		if err != nil {
 			varMap["refresh_status"] = constant.StatusFailed
 			varMap["refresh_msg"] = err.Error()
 			global.LOG.Errorf("Failed to refresh OneDrive token, please retry, err: %v", err)
-			return
+			continue
 		}
-		varMap["refresh_token"] = refreshToken
-
-		varsItem, _ := json.Marshal(varMap)
-		_ = global.DB.Model(&model.BackupAccount{}).
-			Where("id = ?", backupItem.ID).
-			Updates(map[string]interface{}{
-				"vars": varsItem,
-			}).Error
-		global.LOG.Info("Successfully refreshed OneDrive token.")
-	}
-}
-
-func refreshALIYUN() {
-	var backups []model.BackupAccount
-	_ = global.DB.Where("`type` = ?", "ALIYUN").Find(&backups)
-	for _, backupItem := range backups {
-		global.LOG.Infof("start to refresh token of ALIYUN %s ...", backupItem.Name)
-		varMap := make(map[string]interface{})
-		if err := json.Unmarshal([]byte(backupItem.Vars), &varMap); err != nil {
-			global.LOG.Errorf("Failed to refresh ALIYUN token, please retry, err: %v", err)
-			return
-		}
-		if _, ok := varMap["refresh_token"]; !ok {
-			global.LOG.Error("no such refresh token find in db")
-			return
-		}
-		refreshToken, err := client.RefreshALIToken(fmt.Sprintf("%v", varMap["refresh_token"]))
 		varMap["refresh_status"] = constant.StatusSuccess
 		varMap["refresh_time"] = time.Now().Format(constant.DateTimeLayout)
-		if err != nil {
-			varMap["refresh_status"] = constant.StatusFailed
-			varMap["refresh_msg"] = err.Error()
-			global.LOG.Errorf("Failed to refresh ALIYUN token, please retry, err: %v", err)
-			return
-		}
 		varMap["refresh_token"] = refreshToken
 
 		varsItem, _ := json.Marshal(varMap)
-		_ = global.DB.Model(&model.BackupAccount{}).
-			Where("id = ?", backupItem.ID).
-			Updates(map[string]interface{}{
-				"vars": varsItem,
-			}).Error
-		global.LOG.Info("Successfully refreshed ALIYUN token.")
+		_ = global.DB.Model(&model.BackupAccount{}).Where("id = ?", backupItem.ID).Updates(map[string]interface{}{"vars": varsItem}).Error
 	}
 }
```

Comment:
The code you've presented contains a few discrepancies:

  1. An incorrect comment tag at the beginning, `@@`, has been replaced with `- @-`.
  2. The `cron.LoadOneDriveInfo()` function call is incomplete and not properly indented.

There's also some unnecessary commented out blocks of code. To improve readability and maintainability of these repositories, here are suggestions for changes:

1. Remove or document comments that don't contribute to the understanding of the code.
   Original line: 
     ```
@- if account.TokenExpires <= datetime.Today():
```

New suggestion: 

      ```  
if account.TokenExpires < time.Now().AddDate(0, 0, -1).UTC().UnixNano()
```
```


2. Ensure consistent indentation across all lines without adding extra blank spaces before functions like `InitBackupServiceFunc()` which seems redundant given it was previously done elsewhere but wasn't called.
3. Replace missing imports (`import`) statements. Also remove any unused package versions.

These minor revisions should greatly enhance the documentation quality and structure of your repository, making future updates cleaner. Please note that since I'm a text-based AI tool, I can't see how things look on GitHub repos; so feel free to run the latest version locally in my editor and let me know what you think.